### ISSUE TITLE
Update dind to work with Docker 1.7

### ISSUE THREAD

**Issue by tomdee** [jpetazzo/dind#71]
I've tried it with Docker 1.7-rc2.

I can `docker run` new containers.

But I can `docker run --privileged` new containers. I get a cryptic:
```
INFO[0013] POST /v1.19/containers/3836307a2eccb183906dc9591eafe328af980069448890964c3a1dbad5516e31/start 
operation not permitted
WARN[0013] exit status 1                                
ERRO[0013] Handler for POST /containers/{name:.*}/start returned error: Cannot start container 3836307a2eccb183906dc9591eafe328af980069448890964c3a1dbad5516e31: [8] System error: operation not permitted 
ERRO[0013] HTTP Error                                    err=Cannot start container 3836307a2eccb183906dc9591eafe328af980069448890964c3a1dbad5516e31: [8] System error: operation not permitted statusCode=500
Error response from daemon: Cannot start container 3836307a2eccb183906dc9591eafe328af980069448890964c3a1dbad5516e31: [8] System error: operation not permitted
```

**Comment by jpetazzo** [comment]
Do others report the same issue?

**Comment by tomdee** [comment]
This is actually working for me now, so I'm going to close.

**Comment by jpetazzo** [comment]
Thank you!

On Tue, Jun 16, 2015 at 3:40 PM, Tom Denham <notifications@github.com>
wrote:

> Closed #71 <https://github.com/jpetazzo/dind/issues/71>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jpetazzo/dind/issues/71#event-332611972>.
>



-- 
@jpetazzo <https://twitter.com/jpetazzo>
"Is it safe to run applications into containers?"
http://www.slideshare.net/jpetazzo/docker-linux-containers-lxc-and-security
